00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
```

Comment:
(ignoring today's conversation as we are not yet implementing those tags/etc) 

I noticed Lesson 00 has category and the other lessons don't.  
- Do you want it removed from 00 or added to all lessons? 
- same with comment about the workshop tag in tags.  It is not in L00 but is in all other lessons.